### PR TITLE
50806 final ro cs merge release 1.1.2 change default consultations message

### PR DESCRIPTION
**Pull request by csmith-he**
[AB#50806](https://hedev.visualstudio.com/76e71f30-b1ad-438a-a096-89ac98712f4e/_workitems/edit/50806)
[AB#54010](https://hedev.visualstudio.com/76e71f30-b1ad-438a-a096-89ac98712f4e/_workitems/edit/54010)

Replaces PR #275

I had to rework the userCanViewConsultations function that Rob has added to the viewmodel to provide the correct message to the user. There are no params.cards data when viewing in report view which is why the incorrect message was showing even when a user had permissions to view the data.

NB aside from my rework there are also some changes from Rob's original work which I've copied and pasted in as I didn't have access to the working branch and didn't want to deal with merge conflicts from the 1.1.1 branch that was available (basically, I'm saying some of this work is Rob's as well as mine)

The function now uses inputs that are available in report and resource views. Tested locally.

Tester: @Rehaan-Arif
Release Merge Lead: @aj-he

### CODE REVIEW

File: arches_her/media/js/views/components/reports/scenes/resources.js [hunk 1]
```diff
@@ -94,6 +94,73 @@ define(['underscore', 'knockout', 'arches', 'utils/report', 'bindings/datatable'
                     }));
                 }
 
+                const userAvailableConsulationCards = () => {
+                    if(self.dataConfig.resourceinstanceid){
+                        return $.ajax({
+                            url: arches.urls.api_card + self.dataConfig.resourceinstanceid,
+                            context: this,
+                        }).done(function(response) {
+                            console.log(response)
```
Comment (aj-he) [inline arches_her/media/js/views/components/reports/scenes/resources.js:103]: can you remove this console.log

File: arches_her/media/js/views/components/reports/scenes/resources.js [hunk 2]
```diff
@@ -94,6 +94,73 @@ define(['underscore', 'knockout', 'arches', 'utils/report', 'bindings/datatable'
...
+                    }
+                     else {
+                        return false;
+                    }
+                }
+
+
+
+                if(!userCanViewConsultations()) {
+                    userAvailableConsulationCards().then(function(cards_response){
```
Comment (aj-he) [inline arches_her/media/js/views/components/reports/scenes/resources.js:146]: Do you need both these functions? Will the newer `userAvailableConsultationCards` not fulfil the requirement alone?

File: arches_her/media/js/views/components/reports/scenes/resources.js (hunk 1 above)
Comment (csmith-he) [reply arches_her/media/js/views/components/reports/scenes/resources.js]: Removed

File: arches_her/media/js/views/components/reports/scenes/resources.js (hunk 2 above)
Comment (csmith-he) [reply arches_her/media/js/views/components/reports/scenes/resources.js]: Removed userCanViewConsultations()

**Review by aj-he — APPROVED** [review]
Looks good to me

**Pull request merged**